tools/c7n_azure/c7n_azure/resources/postgresql_server.py

```diff
@@ -69,3 +71,77 @@ def _query_rules(self, resource):
                 continue
             resource_rules.add(rule)
         return resource_rules
+
+
+@PostgresqlServer.filter_registry.register('configuration-parameters')
```

Comment:
given an individual use of the filter is on a single attribute, using a singular form for naming the filter seems more appropriate.

Reply:
Agreed, you could technically evaluate multiple params with JMESPath, but it would be awkward.